core/src/test/scala/cromwell/util/AkkaTestUtil.scala

```diff
@@ -19,6 +21,18 @@ object AkkaTestUtil {
         case inbound => probe.ref forward inbound
       }
     })
+
+    @tailrec
+    final def expectMsgAmongstOthers(message: Any, maxWait: FiniteDuration = 10.seconds): Unit = {
+      val waitStart = System.currentTimeMillis()
+      val nextMessage = probe.expectMsgPF(10.seconds) { case anything => anything }
+      nextMessage match {
+        case `message` => () // success!
+        case _ =>
+          val waitedSoFar = System.currentTimeMillis() - waitStart
+          probe.expectMsgAmongstOthers(message, maxWait.minus(waitedSoFar.millis))
+      }
+    }
```

Comment:
a little surprised there isn't something like this already in TestKit but I didn't look too hard 🤷‍♂

Reply:
👍 ... it turns out that it does, it just has a weird name `fishForMessage` that doesn't start with `expect...()` like the others 🤷‍♂